Foundation/NSGeometry.swift

```diff
@@ -11,6 +11,7 @@
     import Darwin
 #elseif os(Linux)
     import Glibc
+    import CoreFoundation
```

Comment:
@phausler https://github.com/apple/swift/blob/master/stdlib/public/Glibc/module.map.in#L51
Seems that this import is commented out intentionally: `note: supplied by compiler`. But for me it doesn't work. I've just updated pull request, reverted `CGFloat` `==` to it's initial state and changed tests, so they do comparison. Could you please try to remove `import CoreFoundation` from the `TestNSGeometry:15` and see if it works for you? May be there's something wrong with my Glibc library?